cdap-docs/developer-manual/source/pipelines/developing-plugins/plugin-basics.rst

```diff
@@ -74,9 +74,7 @@ These annotations are used for plugin classes:
 
 - ``@Description``: Annotation used to add a description of the plugin.
 
-- ``@Requirements``: Annotation used to specify the
-`Requirements <../../reference-manual/javadocs/co/cask/cdap/api/annotation/Requirements.html>`__ needed by a plugin
-to run successfully.
+- ``@Requirements``: Annotation used to specify the `Requirements <../../reference-manual/javadocs/co/cask/cdap/api/annotation/Requirements.html>`__ needed by a plugin to run successfully.
```

Comment:
it works if you split it across lines like this?

Reply:
Yes it does.

https://builds.cask.co/artifact/CDAP-DQB434/shared/build-3/Docs-HTML/5.1.0-SNAPSHOT/en/developer-manual/pipelines/developing-plugins/plugin-basics.html#class-annotations